lang/LangSource/SC_LanguageClient.cpp

```diff
@@ -99,8 +99,9 @@ void SC_LanguageClient::initRuntime(const Options& opt) {
     // start virtual machine
     if (!mHiddenClient->mRunning) {
         mHiddenClient->mRunning = true;
-        if (opt.mRuntimeDir) {
-            int err = chdir(opt.mRuntimeDir);
+        if (opt.mRuntimeDir.data() != nullptr) {
+            // int err = chdir(opt.mRuntimeDir);
+            int err = 0;
```

Comment:
This needs a comment. Why did you remove the `chdir` call? If you remove it, then you should also remove the subsequent error check.

If there is a good reason not to switch the current working directory, then you should put the whole `if` clause in an `#if 0 ` block (+ add a comment why the code is disabled)

Reply:
This was actually a comment during development - reintroduced the functionality, thx for spotting it!